Competition2017/src/competition/subsystems/shoot_fuel/FireLeftTracerRoundsCommandGroup.java

```diff
@@ -0,0 +1,29 @@
+package competition.subsystems.shoot_fuel;
+
+import com.google.inject.Inject;
+
+import competition.subsystems.agitator.AgitatorsManagerSubsystem;
+import competition.subsystems.agitator.commands.IntakeAgitatorCommand;
+import competition.subsystems.shooter_belt.ShooterBeltsManagerSubsystem;
+import competition.subsystems.shooter_belt.commands.RunBeltSlowlyUsingPowerCommand;
+import competition.subsystems.shooter_wheel.ShooterWheelsManagerSubsystem;
+import edu.wpi.first.wpilibj.command.CommandGroup;
+
+public class FireLeftTracerRoundsCommandGroup extends CommandGroup {
```

Comment:
Actually, you might want to take a look at `IntakeAgitatorCommand ` to see how that is done.